extensions/amp-analytics/amp-video-analytics.md

```diff
@@ -0,0 +1,142 @@
+"triggers": {
+  "myVideoSession": {
+    "on": "video-session",
+    "request": "event"
+  },
+  "videoSpec": {/* optional videoSpec */}
+}
+```
+
+### Video seconds played trigger (`"on": "video-seconds-played"`)
+
+The `video-seconds-played` trigger is fired when a specified number of seconds have passed since the last time the event fired. This *requires* `interval` to be set in the `videoSpec`. Use these configurations to fire a request for this event.
+
+```javascript
+"triggers": {
+  "myVideoSecondsPlayed": {
+    "on": "video-seconds-played",
+    "request": "event"
+  },
+  "videoSpec": {
+    "interval": 10, /* required */
+    /* other optional videoSpec properties */
+  }
+}
+```
+
+
+## Video spec
+
+You can use the `videoSpec` configuration object to control how the requests are fired.
+
+### `end-session-when-invisible`
+
+This spec property controls the `video-session` trigger. If true, the trigger will fire when the video is no longer visible, even if it is still playing. The default value is `false`.
+
+### `exclude-autoplay`
+
+This spec property controls all `video-*` triggers. If true, the trigger will not fire for videos that are currently autoplaying, not just initially autoplaying. If a user interacts with the video and changes its state from autoplaying, the trigger will include events issued by the video. The default value is `false`.
+
+### `interval`
+
+This spec property controls the `video-seconds-played` trigger, and it is *required* for this trigger. Its value is the number of seconds between each `video-seconds-played` event. It must be greater than `0`. For example, a value of `10` would fire the trigger every 10 seconds.
+
+
+## Common variables
+
+All video analytics triggers expose the following variables. These variables are also available in variable substitution with the [`VIDEO_STATE`](#VIDEO_STATE) variable.
+
+| Var | Type | Description |
+|--|--|--|
+| ${autoplay} | Boolean | Indicates whether the video began as an autoplay video. |
+| ${currentTime} | Number | Specifies the current playback time (in seconds) at the time of trigger. |
+| ${duration} | Number | Specifies the total duration of the video (in seconds). |
+| ${height} | Number | Specifies the height of video (in px). |
+| ${id} | String | Specifies the ID of the video element. |
+| ${playedTotal} | Number | Specifies the total amount of time the user has watched the video. |
+| ${state} | String | Indicates the state, which can be one “playing_auto”, “playing_manual”, or “paused”. |
+| ${width} | Number | Specifies the width of video (in px). |
+| ${playedRangesJson} | String | Representssegments of time the user has watch the video (in JSON format). For example, [[1, 10], [5, 20]] |
+| ${playedTotal} | Number | Specifies the total playing time for the session (in seconds). |
+
+
+## Video analytics variables
+
+Video analytics contributes the following URL substitutions to [amp-var-substitutions.md](../../spec/amp-var-substitutions.md).
```

Comment:
Replace "amp-var-substitutions.md" with the actual title of the document.

Reply:
I don't think so, but I'm open to other views. I based this document off of `amp-access-analytics.md` which also exposes url substitution variables that are not repeated in `amp-var-substitutions.md`.